src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx

```diff
@@ -12,45 +12,53 @@ import AppointmentDateTime from './cards/confirmed/AppointmentDateTime';
 import { getVARFacilityId } from '../../services/appointment';
 import AppointmentInstructions from './cards/confirmed/AppointmentInstructions';
 import AddToCalendar from '../../components/AddToCalendar';
-import { selectFeatureCancel } from '../../redux/selectors';
 import FacilityAddress from '../../components/FacilityAddress';
 import { formatFacilityAddress } from '../../services/location';
+import PageLayout from './AppointmentsPage/PageLayout';
+import ErrorMessage from '../../components/ErrorMessage';
+import { selectConfirmedAppointmentById } from '../redux/selectors';
+import FullWidthLayout from '../../components/FullWidthLayout';
 
 function CommunityCareAppointmentDetailsPage({
-  appointmentDetails,
+  appointment,
   appointmentDetailsStatus,
   fetchConfirmedAppointmentDetails,
-  confirmedStatus,
 }) {
   const { id } = useParams();
-  const history = useHistory();
-
-  const appointment = appointmentDetails?.[id];
 
   useEffect(() => {
-    const status = confirmedStatus === FETCH_STATUS.succeeded;
-
-    if (!status) {
-      history.push('/');
+    if (!appointment) {
```

Comment:
Made some minimal tweaks in here to keep up with other changes, but this page doesn't support refreshing right now.